portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactsP8nInfo.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 The Data-Portability Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataportabilityproject.serviceProviders.google.contacts;
+
+import com.google.common.base.MoreObjects;
+import org.dataportabilityproject.dataModels.PaginationInformation;
+
+public class GoogleContactsP8nInfo implements PaginationInformation {
```

Comment:
Cool acronym, I hadn't seen P8n before. Looks like we using the full "Pagination" everywhere else though, so can you do that here too to be consistent?

Reply:
GoogleContactsServicePaginationInfo is 35 characters long, and was getting kinda unwieldy.  Perhaps GContactsPaginationInfo?

Reply:
how about GooglePaginationInfo and move it to shared? I have a feeling Google APIs will use consistent paging info, if not we can cross that bridge when we get there